src/Illuminate/Auth/DatabaseUserProvider.php

```diff
@@ -122,7 +122,7 @@ public function retrieveByCredentials(array $credentials)
             }
         }
 
-        // Now we are ready to execute the query to see if we have an user matching
+        // Now we are ready to execute the query to see if we have a user matching
```

Comment:
I'm not a native English speaker but this change looks wrong to me.

https://www.grammarly.com/blog/indefinite-articles-a-and-an/

> Use **a** when the indefinite article comes before a word beginning with a consonant sound
 Use **an** when the indefinite article comes before a word beginning with a vowel sound

Reply:
Unfortunately the English language can be complicated with various exceptions to the rule.

[This article](https://www.quora.com/Is-it-an-user-or-a-user) has several explanations but I think this one sums things up well.

> _The use of the indefinite articles in English: "a" and "an"—which one of them to use—depends on the first sound of the word that will follow the article. For example, the word "user" is pronounced "yoozer" (the initial "y" sound acts as a consonant) so it must be preceded by an "a," not an "an." Now if the word were "usher," which does not begin with a consonantal sound, the correct article to use would be "an."_